### PR TITLE
feat(react-linter): extend config with sort import alphabet keys and better prettier integration

### PR DESCRIPTION
**Pull request by kwiss**
extend config with sort import alphabet keys and better prettier integration

### CODE REVIEW

File: packages/eslint-config-react/index.js [hunk 1]
```diff
@@ -1,15 +1,37 @@
+const error = 2;
+const warn = 1;
+const ignore = 0;
+
 module.exports = {
   extends: [
     "@socialgouv/eslint-config-recommended",
+    "airbnb",
```
Comment (douglasduteil) [inline packages/eslint-config-react/index.js]: not react related

File: packages/eslint-config-react/index.js [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
     "plugin:jsx-a11y/recommended",
     "plugin:react/recommended",
-    "prettier/react"
+    "prettier",
```
Comment (douglasduteil) [inline packages/eslint-config-react/index.js]: already in "@socialgouv/eslint-config-recommended"

File: packages/eslint-config-react/index.js [hunk 3]
```diff
@@ -1,15 +1,37 @@
...
+    "prettier/react",
+    "plugin:import/errors",
```
Comment (douglasduteil) [inline packages/eslint-config-react/index.js]: not react related

File: packages/eslint-config-react/index.js [hunk 4]
```diff
@@ -1,15 +1,37 @@
...
+    "plugin:import/warnings"
```
Comment (douglasduteil) [inline packages/eslint-config-react/index.js]: not react related

File: packages/eslint-config-react/index.js [hunk 5]
```diff
@@ -1,15 +1,37 @@
...
   ],
   parser: "babel-eslint",
-  plugins: ["jsx-a11y", "react", "react-hooks"],
+  plugins: [
+    "jsx-a11y",
+    "react",
+    "react-hooks",
+    "sort-keys-fix",
```
Comment (douglasduteil) [inline packages/eslint-config-react/index.js]: not react related

**Comment by douglasduteil** [comment]
This will impact **all our js projects** so I'm adding everyone here

**Comment by lionelB** [comment]
does `prettier --fix` handle that for us ?

File: packages/eslint-config-react/index.js [hunk 6]
```diff
@@ -1,15 +1,37 @@
+const error = 2;
+const warn = 1;
+const ignore = 0;
```
Comment (lionelB) [inline packages/eslint-config-react/index.js]: I thought you like alphabetical order 🤔

**Comment by lionelB** [comment]
It would be nice to have a little description about what all these new plugins are usefull for !

File: packages/eslint-config-react/index.js (hunk 6 above)
Comment (kwiss) [reply packages/eslint-config-react/index.js]: It was late at night ^^

File: packages/eslint-config-react/index.js (hunk 1 above)
Comment (kwiss) [reply packages/eslint-config-react/index.js]: Ok Wrong package total fail I’ll use Airbnb-react instead

**Comment by kwiss** [comment]
> This will impact **all our js projects** so I'm adding everyone here
@douglasduteil 
maybe it's too much but should we instead create an additional/optionnal package ?

**Comment by revolunet** [comment]
wadzup

**Pull request closed without merging**